pom.xml

```diff
@@ -62,6 +62,10 @@
     <jacoco.agent.argLine/>
     <sonar.projectKey>optaweb-employee-rostering</sonar.projectKey>
     <sonar.moduleKey>${project.artifactId}</sonar.moduleKey>
+    <sonar.test.inclusions>
+      optaweb-employee-rostering-frontend/**/*test.ts,
+      optaweb-employee-rostering-frontend/**/*test.tsx
+    </sonar.test.inclusions>
```

Comment:
We can't move it to the parent as that would affect other modules. This would result in excluding Java test files from scanning (note that's something different from coverage analysis).